AzureCalling/Controllers/BottomDrawerViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+protocol BottomDrawerViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
Protocol is usually defined for an interface but not the some generation method. It doesn't contain the implement of the things. I dont think it is the best choice to define BottomDrawerViewController as a protocol even it work here. At least we should define it as a base viewcontroller, and participant list and audio selection drawer can inherit from it. 
Ideally, could the bottomDrawerViewController be the only drawer, and we input different datasource of BottomDrawerItem into the drawer? Maybe the bottomDrawerViewController has 2 different style type, one is for AudioSelection, one is for ParticipantList to build the view difference